pkg/logger/logger.go

```diff
@@ -0,0 +1,59 @@
+package logger
+
+import (
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	logger *zap.Logger
```

Comment:
I think it's not good practice to use global variables

Reply:
in most cases I agree but take a look at my reply https://github.com/archway-network/validator-exporter/pull/1#discussion_r1432851734